dev/com.ibm.ws.concurrent_fat_jakarta_11/fat/src/test/jakarta/concurrency/Concurrency31Test.java

```diff
@@ -33,7 +33,7 @@
 import test.jakarta.concurrency31.web.Concurrency31TestServlet;
 
 @RunWith(FATRunner.class)
-@MinimumJavaLevel(javaLevel = 21)
+@MinimumJavaLevel(javaLevel = 17)
```

Comment:
I don't think some of the tests in this bucket will pass on Java SE 17 without updates.

Reply:
Yeah, I was testing locally using java 21.  I reverted this change.